### PR TITLE
Replacement of font size variables

### PR DESCRIPTION
**Pull request by GusBaamonde**
I removed the old font size variables, replacing them with new variables in levels, impacting all the scss files. I've talked to Santiago Paolucci and he agrees.

variables-font-size

### CODE REVIEW

File: assets/scss/core/_variables.scss [hunk 1]
```diff
@@ -25,26 +26,33 @@ $dp-font-color-dark: $dp-font-color !default;
...
-$dp-font-button-medium: 13px;
-$dp-font-button-small: 11px;
-$dp-font-size-15: 15px;
-$dp-font-size-24: 24px;
-$dp-font-small-lineheight: 18px;
+////
+/// Font Size settings
+///
+
+$dp-font-size--lv0: 0;
```
Comment (andresmoschini) [inline assets/scss/core/_variables.scss]: Does this variable make sense?

**Comment by andresmoschini** [comment]
It is fine for me, maybe @mvillaseco would like to review it.

File: assets/scss/core/_variables.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/core/_variables.scss]: En algunos casos necesitamos ocultar el contenido, actualmente pasa con el logo en signup, el navegador lo lee pero no tiene tamaño el contenido y se coloca la imagen por background

Comment (andresmoschini) [reply assets/scss/core/_variables.scss]: Thanks for the explanation, you are the expert.

File: assets/scss/core/_variables.scss [hunk 2]
```diff
@@ -25,26 +26,33 @@ $dp-font-color-dark: $dp-font-color !default;
...
+$dp-font-size--lv1: 10px;
+$dp-font-size--lv2: 11px;
+$dp-font-size--lv3: 12px;
+$dp-font-size--lv4: 13px;
+$dp-font-size--lv5: 14px;
+$dp-font-size--lv6: 15px;
```
Comment (cbernat) [inline assets/scss/core/_variables.scss]: Do we use all this fontsizes for real? Seems a bit too much

Comment (mvillaseco) [reply assets/scss/core/_variables.scss]: Omg, 19 font sizes? are we using them as other things also?(maybe a rename should be done)

**Comment by damianmuti** [comment]
Despite the fact that we all agreed to have levels for sizes, it seems that this approach lacks of any sort of description in the naming.
Just as an exercise for anyone that looks at this comment, try to think about what's the font size for buttons with this new naming convention (without scrolling up). No clue? Me neither. 😝

File: assets/scss/core/_variables.scss (hunk 2 above)
Comment (andresmoschini) [reply assets/scss/core/_variables.scss]: I agree as we have standardized spaces for common usages, we should have standardized font sizes for common usages.

**Comment by IgnacioRodrigues** [comment]
This nomenclature of variables seems not to be the correct one, like knowing what level to use in each case? Support to what they say in which the nomenclatures tell you to use them, with this change you lose the sense of using classes.

**Comment by GusBaamonde** [comment]
fonts


**To unify criteria of variables, I talked with Santiago Paolucci, @mvillaseco  and @damianmuti and I think we reached a midpoint.**

- I made a survey of the size fonts currently used.
- I reduced the quantity of size fonts used.
- We will use the guide already posed in the mockup of the library for future designs (we talk with Santiago Paolucci and compare with the fonts of the mockup of the library).
- We will use more global levels (We speak with @mvillaseco ).
- We will not have specific names at the moment. (We talked to @damianmuti , and it's a good idea to make 'variables of variables' in order to make themes, improve for the future)
- I changed and cleaned all the font size variables we are using.

**Comment by andresmoschini** [comment]
Looks fine!

**Pull request merged**